llvm_passes/HipAbort.cpp

```diff
@@ -184,8 +184,14 @@ void HipAbortPass::analyze(const CallGraph &CG) {
   }
 
   // Propagate may-abort attribute to callers.
+
   while (WorkList.size()) {
+    printf("worklist size: %d\n", WorkList.size());
     InverseCallGraphNode *N = popAny(WorkList);
+    if (N == nullptr) {
+      printf("N was nullptr! skipping\n");
```

Comment:
This gets triggered in the `abort` sample. Why?
Is this expected or a macOS issue?